.github/workflows/testing.yml

```diff
@@ -91,6 +84,17 @@ jobs:
           pip install ${{ matrix.msgpack-deps }}
           sed -i -e "s/^msgpack.*$/${{ matrix.msgpack-deps }}/" requirements.txt
 
+      - name: Install specific version of msgpack package
+        # We want to enforce using modern msgpack since it has
+        # various vulnerability fixes. But the code is compatible
+        # with older msgpack versions. To this test compatibility
+        # we must ignore requirements.txt install of the newer msgpack
+        # package by overwriting it with sed.
+        if: startsWith(matrix.msgpack-deps, 'msgpack==') == true
+        run: |
+          pip install ${{ matrix.msgpack-deps }}
+          sed -i -e "s/^msgpack.*$/${{ matrix.msgpack-deps }}/" requirements.txt
+
```

Comment:
I don't understand why it is necessary twice. A condition lools the same:
```yaml
        if: startsWith(matrix.msgpack-deps, 'msgpack==') == true
```

```yaml
# https://github.com/tarantool/tarantool-python/blob/DifferentialOrange/update-deps/.github/workflows/testing.yml
      - name: Install specific version of msgpack package
        if: startsWith(matrix.msgpack-deps, 'msgpack==') == true
        run: |
          pip install ${{ matrix.msgpack-deps }}
...
      - name: Install specific version of msgpack package
        # We want to enforce using modern msgpack since it has
        # various vulnerability fixes. But the code is compatible
        # with older msgpack versions. To this test compatibility
        # we must ignore requirements.txt install of the newer msgpack
        # package by overwriting it with sed.
        if: startsWith(matrix.msgpack-deps, 'msgpack==') == true
        run: |
          pip install ${{ matrix.msgpack-deps }}
          sed -i -e "s/^msgpack.*$/${{ matrix.msgpack-deps }}/" requirements.txt
```
Maybe one of that needs to be deleted?

Reply:
Also I think it could be merged:
```
      - name: Install specific version of msgpack-python package
        # msgpack package is a replacement for deprecated msgpack-python.
        # To test compatibility with msgpack-python we must ignore
        # requirements.txt install of msgpack package by overwriting it
        # with sed.
        if: startsWith(matrix.msgpack-deps, 'msgpack-python==') == true
        run: |
          pip install ${{ matrix.msgpack-deps }}
          sed -i -e "s/^msgpack.*$/${{ matrix.msgpack-deps }}/" requirements.txt
      - name: Install specific version of msgpack package
        # We want to enforce using modern msgpack since it has
        # various vulnerability fixes. But the code is compatible
        # with older msgpack versions. To this test compatibility
        # we must ignore requirements.txt install of the newer msgpack
        # package by overwriting it with sed.
        if: startsWith(matrix.msgpack-deps, 'msgpack==') == true
        run: |
          pip install ${{ matrix.msgpack-deps }}
          sed -i -e "s/^msgpack.*$/${{ matrix.msgpack-deps }}/" requirements.txt
```
into a one
```
      - name: Install specific version of msgpack-python package
        # msgpack package is a replacement for deprecated msgpack-python.
        # To test compatibility with msgpack-python we must ignore
        # requirements.txt install of msgpack package by overwriting it
        # with sed.
        if: | # or something like that, I don't too good with yaml-syntax
          startsWith(matrix.msgpack-deps, 'msgpack-python==') == true ||
          startsWith(matrix.msgpack-deps, 'msgpack==') == true
        run: |
          pip install ${{ matrix.msgpack-deps }}
          sed -i -e "s/^msgpack.*$/${{ matrix.msgpack-deps }}/" requirements.txt
```

Reply:
Also it should be synchronized with the `run_tests_ee_linux` workflow.

Reply:
To be honest, the only reason these steps are different is so they will have different names... but this is really not matters. Unified them into one.

> Also the changes should be synchronized with the run_tests_ee_linux workflow.
Tarantool EE is tested only with msgpack from the requirements (latest one), so the variable and steps here are not even used there. I have removed them from a pipeline. (I don't see a reason to add different msgpacks here because there are already too many pipelines.)